e2e/test/scenarios/native-filters/helpers/e2e-field-filter-helpers.js

```diff
@@ -36,7 +36,7 @@ export function clearWidgetValue() {
 }
 
 export function setWidgetStringFilter(value) {
-  popover().find("input").first().type(`${value}{enter}`);
+  popover().find("input").not("[type=hidden]").first().type(`${value}{enter}`);
```

Comment:
We cannot use the `multiAutocompleteInput` helper here because we don't know what type of input will be rendered per se. This selector works for both normal inputs and the `MultiAutocomplete` input.